# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# 
# Translators:
# Zoltan Szalai <defaultdict@gmail.com>, 2015
msgid ""
msgstr ""
"Project-Id-Version: Django REST framework\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2015-06-03 17:30+0100\n"
"PO-Revision-Date: 2015-06-03 16:30+0000\n"
"Last-Translator: Xavier Ordoquy <xordoquy@linovia.com>\n"
"Language-Team: Hungarian (http://www.transifex.com/projects/p/django-rest-framework/language/hu/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Language: hu\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: authentication.py:70
msgid "Invalid basic header. No credentials provided."
msgstr "Érvénytelen basic fejlécmező. Nem voltak megadva azonosítók."

#: authentication.py:73
msgid "Invalid basic header. Credentials string should not contain spaces."
msgstr "Érvénytelen basic fejlécmező. Az azonosító karakterlánc nem tartalmazhat szóközöket."

#: authentication.py:79
msgid "Invalid basic header. Credentials not correctly base64 encoded."
msgstr "Érvénytelen basic fejlécmező. Az azonosítók base64 kódolása nem megfelelő."

#: authentication.py:97
msgid "Invalid username/password."
msgstr "Érvénytelen felhasználónév/jelszó."

#: authentication.py:100 authentication.py:182
msgid "User inactive or deleted."
msgstr "A felhasználó nincs aktiválva vagy törölve lett."

#: authentication.py:167
msgid "Invalid token header. No credentials provided."
msgstr "Érvénytelen token fejlécmező. Nem voltak megadva azonosítók."

#: authentication.py:170
msgid "Invalid token header. Token string should not contain spaces."
msgstr "Érvénytelen token fejlécmező. A token karakterlánc nem tartalmazhat szóközöket."

#: authentication.py:179
msgid "Invalid token."
msgstr "Érvénytelen token."

#: authtoken/serializers.py:20
msgid "User account is disabled."
msgstr "A felhasználó tiltva van."

#: authtoken/serializers.py:23
msgid "Unable to log in with provided credentials."
msgstr "A megadott azonosítókkal nem lehet bejelentkezni."

#: authtoken/serializers.py:26
msgid "Must include \"username\" and \"password\"."
msgstr "Tartalmaznia kell a \"felhasználónevet\" és a \"jelszót\"."

#: exceptions.py:38
msgid "A server error occurred."
msgstr "Szerver oldali hiba történt."

#: exceptions.py:73
msgid "Malformed request."
msgstr "Hibás kérés."

#: exceptions.py:78
msgid "Incorrect authentication credentials."
msgstr "Hibás azonosítók."

#: exceptions.py:83
msgid "Authentication credentials were not provided."
msgstr "Nem voltak megadva azonosítók."

#: exceptions.py:88
msgid "You do not have permission to perform this action."
msgstr "Nincs jogosultsága a művelet végrehajtásához."

#: exceptions.py:93 views.py:78
msgid "Not found."
msgstr "Nem található."

#: exceptions.py:98
#, python-brace-format
msgid "Method \"{method}\" not allowed."
msgstr "A \"{method}\" metódus nem megengedett."

#: exceptions.py:109
msgid "Could not satisfy the request Accept header."
msgstr "A kérés Accept fejlécmezőjét nem lehetett kiszolgálni."

#: exceptions.py:121
#, python-brace-format
msgid "Unsupported media type \"{media_type}\" in request."
msgstr "Nem támogatott média típus \"{media_type}\" a kérésben."

#: exceptions.py:134
msgid "Request was throttled."
msgstr "A kérés korlátozva lett."

#: fields.py:162 relations.py:132 relations.py:156 validators.py:77
#: validators.py:160
msgid "This field is required."
msgstr "Ennek a mezőnek a megadása kötelező."

#: fields.py:163
msgid "This field may not be null."
msgstr "Ez a mező nem lehet null értékű."

#: fields.py:496 fields.py:524
#, python-brace-format
msgid "\"{input}\" is not a valid boolean."
msgstr "Az \"{input}\" nem egy érvényes logikai érték."

#: fields.py:559
msgid "This field may not be blank."
msgstr "Ez a mező nem lehet üres."

#: fields.py:560 fields.py:1386
#, python-brace-format
msgid "Ensure this field has no more than {max_length} characters."
msgstr "Bizonyosodjon meg arról, hogy ez a mező legfeljebb {max_length} karakterből áll."

#: fields.py:561
#, python-brace-format
msgid "Ensure this field has at least {min_length} characters."
msgstr "Bizonyosodjon meg arról, hogy ez a mező legalább {min_length} karakterből áll."

#: fields.py:598
msgid "Enter a valid email address."
msgstr "Adjon meg egy érvényes e-mail címet!"

#: fields.py:609
msgid "This value does not match the required pattern."
msgstr "Ez az érték nem illeszkedik a szükséges mintázatra."

#: fields.py:620
msgid ""
"Enter a valid \"slug\" consisting of letters, numbers, underscores or "
"hyphens."
msgstr "Az URL barát cím csak betűket, számokat, aláhúzásokat és kötőjeleket tartalmazhat."

#: fields.py:632
msgid "Enter a valid URL."
msgstr "Adjon meg egy érvényes URL-t!"

#: fields.py:645
#, python-brace-format
msgid "\"{value}\" is not a valid UUID."
msgstr ""

#: fields.py:679
msgid "A valid integer is required."
msgstr "Egy érvényes egész szám megadása szükséges."

#: fields.py:680 fields.py:715 fields.py:748
#, python-brace-format
msgid "Ensure this value is less than or equal to {max_value}."
msgstr "Bizonyosodjon meg arról, hogy ez az érték legfeljebb {max_value}."

#: fields.py:681 fields.py:716 fields.py:749
#, python-brace-format
msgid "Ensure this value is greater than or equal to {min_value}."
msgstr "Bizonyosodjon meg arról, hogy ez az érték legalább {min_value}."

#: fields.py:682 fields.py:717 fields.py:753
msgid "String value too large."
msgstr "A karakterlánc túl hosszú."

#: fields.py:714 fields.py:747
msgid "A valid number is required."
msgstr "Egy érvényes szám megadása szükséges."

#: fields.py:750
#, python-brace-format
msgid "Ensure that there are no more than {max_digits} digits in total."
msgstr "Bizonyosodjon meg arról, hogy a számjegyek száma összesen legfeljebb {max_digits}."

#: fields.py:751
#, python-brace-format
msgid ""
"Ensure that there are no more than {max_decimal_places} decimal places."
msgstr "Bizonyosodjon meg arról, hogy a tizedes tört törtrészében levő számjegyek száma összesen legfeljebb {max_decimal_places}."

#: fields.py:752
#, python-brace-format
msgid ""
"Ensure that there are no more than {max_whole_digits} digits before the "
"decimal point."
msgstr "Bizonyosodjon meg arról, hogy a tizedes tört egész részében levő számjegyek száma összesen legfeljebb {max_whole_digits}."

#: fields.py:842
#, python-brace-format
msgid "Datetime has wrong format. Use one of these formats instead: {format}."
msgstr "A dátum formátuma hibás. Használja ezek valamelyikét helyette: {format}."

#: fields.py:843
msgid "Expected a datetime but got a date."
msgstr "Időt is tartalmazó dátum helyett egy időt nem tartalmazó dátum lett elküldve."

#: fields.py:907
#, python-brace-format
msgid "Date has wrong format. Use one of these formats instead: {format}."
msgstr "A dátum formátuma hibás. Használja ezek valamelyikét helyette: {format}."

#: fields.py:908
msgid "Expected a date but got a datetime."
msgstr "Időt nem tartalmazó dátum helyett egy időt is tartalmazó dátum lett elküldve."

#: fields.py:971
#, python-brace-format
msgid "Time has wrong format. Use one of these formats instead: {format}."
msgstr "Az idő formátuma hibás. Használja ezek valamelyikét helyette: {format}."

#: fields.py:1025
#, python-brace-format
msgid "Duration has wrong format. Use one of these formats instead: {format}."
msgstr ""

#: fields.py:1050 fields.py:1094
#, python-brace-format
msgid "\"{input}\" is not a valid choice."
msgstr "Az \"{input}\" nem egy érvényes elem."

#: fields.py:1095 fields.py:1213 serializers.py:487
#, python-brace-format
msgid "Expected a list of items but got type \"{input_type}\"."
msgstr "Elemek listája helyett \"{input_type}\" lett elküldve."

#: fields.py:1129
msgid "No file was submitted."
msgstr "Semmilyen fájl sem került feltöltésre."

#: fields.py:1130
msgid ""
"The submitted data was not a file. Check the encoding type on the form."
msgstr "Az elküldött adat nem egy fájl volt. Ellenőrizze a kódolás típusát az űrlapon!"

#: fields.py:1131
msgid "No filename could be determined."
msgstr "A fájlnév nem megállapítható."

#: fields.py:1132
msgid "The submitted file is empty."
msgstr "A küldött fájl üres."

#: fields.py:1133
#, python-brace-format
msgid ""
"Ensure this filename has at most {max_length} characters (it has {length})."
msgstr "Bizonyosodjon meg arról, hogy a fájlnév legfeljebb {max_length} karakterből áll (jelenlegi hossza: {length})."

#: fields.py:1175
msgid ""
"Upload a valid image. The file you uploaded was either not an image or a "
"corrupted image."
msgstr "Töltsön fel egy érvényes képfájlt! A feltöltött fájl nem kép volt, vagy megsérült."

#: fields.py:1250
#, python-brace-format
msgid "Expected a dictionary of items but got type \"{input_type}\"."
msgstr ""

#: pagination.py:231
#, python-brace-format
msgid "Invalid page \"{page_number}\": {message}."
msgstr "Érvénytelen oldal \"{page_number}\": {message}."

#: pagination.py:492
msgid "Invalid cursor"
msgstr ""

#: relations.py:133
#, python-brace-format
msgid "Invalid pk \"{pk_value}\" - object does not exist."
msgstr "Érvénytelen pk \"{pk_value}\" - az objektum nem létezik."

#: relations.py:134
#, python-brace-format
msgid "Incorrect type. Expected pk value, received {data_type}."
msgstr "Helytelen típus. pk érték helyett {data_type} lett elküldve."

#: relations.py:157
msgid "Invalid hyperlink - No URL match."
msgstr "Érvénytelen link - Nem illeszkedő URL."

#: relations.py:158
msgid "Invalid hyperlink - Incorrect URL match."
msgstr "Érvénytelen link. - Eltérő URL illeszkedés."

#: relations.py:159
msgid "Invalid hyperlink - Object does not exist."
msgstr "Érvénytelen link - Az objektum nem létezik."

#: relations.py:160
#, python-brace-format
msgid "Incorrect type. Expected URL string, received {data_type}."
msgstr "Helytelen típus. URL karakterlánc helyett {data_type} lett elküldve."

#: relations.py:302
#, python-brace-format
msgid "Object with {slug_name}={value} does not exist."
msgstr "Nem létezik olyan objektum, amelynél {slug_name}={value}."

#: relations.py:303
msgid "Invalid value."
msgstr "Érvénytelen érték."

#: serializers.py:304
#, python-brace-format
msgid "Invalid data. Expected a dictionary, but got {datatype}."
msgstr "Érvénytelen adat. Egy dictionary helyett {datatype} lett elküldve."

#: templates/rest_framework/horizontal/radio.html:2
#: templates/rest_framework/inline/radio.html:2
#: templates/rest_framework/vertical/radio.html:2
msgid "None"
msgstr ""

#: templates/rest_framework/horizontal/select_multiple.html:2
#: templates/rest_framework/inline/select_multiple.html:2
#: templates/rest_framework/vertical/select_multiple.html:2
msgid "No items to select."
msgstr ""

#: validators.py:22
msgid "This field must be unique."
msgstr "Ennek a mezőnek egyedinek kell lennie."

#: validators.py:76
#, python-brace-format
msgid "The fields {field_names} must make a unique set."
msgstr "A {field_names} mezőnevek nem tartalmazhatnak duplikátumot."

#: validators.py:224
#, python-brace-format
msgid "This field must be unique for the \"{date_field}\" date."
msgstr "A mezőnek egyedinek kell lennie a \"{date_field}\" dátumra."

#: validators.py:239
#, python-brace-format
msgid "This field must be unique for the \"{date_field}\" month."
msgstr "A mezőnek egyedinek kell lennie a \"{date_field}\" hónapra."

#: validators.py:252
#, python-brace-format
msgid "This field must be unique for the \"{date_field}\" year."
msgstr "A mezőnek egyedinek kell lennie a \"{date_field}\" évre."

#: versioning.py:39
msgid "Invalid version in \"Accept\" header."
msgstr "Érvénytelen verzió az \"Accept\" fejlécmezőben."

#: versioning.py:70 versioning.py:112
msgid "Invalid version in URL path."
msgstr "Érvénytelen verzió az URL elérési útban."

#: versioning.py:141
msgid "Invalid version in hostname."
msgstr "Érvénytelen verzió a hosztnévben."

#: versioning.py:163
msgid "Invalid version in query parameter."
msgstr "Érvénytelen verzió a lekérdezési paraméterben."

#: views.py:85
msgid "Permission denied."
msgstr ""
